distiller/fac_scraper/spiders/reports.py

```diff
@@ -14,7 +14,7 @@
 class FACSpider(Spider):
     name = 'fac'
     start_urls = ['https://harvester.census.gov/facdissem/SearchA133.aspx']
-    download_delay = 1.5
+    #download_delay = 1.5
```

Comment:
Why is this commented out?

Reply:
The delay is a wait time between requests, in seconds. Since we're only doing one crawl at a time, I didn't think we needed to pause between requests. I'll add a comment explaining it.